backend/fake.go

```diff
@@ -23,13 +24,15 @@ func newFakeProvider(cfg *config.ProviderConfig) (Provider, error) {
 }
 
 func (p *fakeProvider) Start(ctx context.Context, _ *StartAttributes) (Instance, error) {
-	return &fakeInstance{p: p}, nil
+	return &fakeInstance{p: p, startupDuration: 42170 * time.Millisecond}, nil
```

Comment:
@moul Confirm, it is.  Injecting it in the tests is probs better.  I'll fix. :heart_eyes_cat: